index.html

```diff
@@ -917,6 +933,22 @@ <h3>
           </li>
         </ol>
       </li>
+      <li>If the session's [=next wallet behavior=] is set:
```

Comment:
As this is duplicative, maybe move the step to its own algorithm? then we only need to maintain one set of steps.